app/src/main/java/com/rarilabs/rarime/manager/VotingManager.kt

```diff
@@ -223,6 +230,12 @@ class VotingManager @Inject constructor(
             else -> "-"
         }
 
+        val expirationDateString = when (decodedMinExpirationDate) {
+            "000000" -> "-"
```

Comment:
you use it only once, and it's never called when decodedMinExpirationDate is zero date, so just move `"Valid until ${decodedMinExpirationDate}"` to PollCriteria title